infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuInterClusterReplicationServer.java

```diff
@@ -271,6 +272,9 @@ private void startServer() {
         }
 
         log.info("main: Server exiting due to shutdown");
+        // Flush the async appender before exiting to prevent the loss of logs
```

Comment:
shouldnt it be added in cleanShutdown?   Just like how it is in CorfuServer?

Reply:
good point. thanks. added there also.

Reply:
keeping it here, in case we shutdown due to some exception

Reply:
shouldnt we also add it in the catch block of startServer()?  You can do it in a follow-up patch if you like.